README.md

```diff
@@ -103,6 +123,8 @@ Quickstart guide
     brew install proper
     ```
     and continue following the instructions below.
+    > The path that you will need, in most cases, if you used `brew` is
+    > `/usr/local/Cellar/proper/1.3`
```

Comment:
Please don't recommend users rely on `Cellar` paths -- these are fragile and could be changed at any moment (i.e. even if proper's version does not change). Instead, users should use what's recommended in `brew info proper`:
```
==> Caveats
To use PropEr in Erlang, you may need:
  export ERL_LIBS=/opt/homebrew/opt/proper/proper-1.4
```